tsconfig.json

```diff
@@ -27,6 +27,6 @@
     "src/**/*"
   ],
   "exclude": [
-    "src/*/files/**/*"
+    "src/*/files/**/*",
```

Comment:
trailing comma

Reply:
Trailing commas is great thing in arrays already many years. Check any modern language. I'm always you them in projects

Reply:
Done

Reply:
this project is written without using trailing comma, so let's not break the code style of this project